packages/plugin-console-breadcrumbs/console-breadcrumbs.js

```diff
@@ -13,7 +13,9 @@ exports.load = (client) => {
   map(CONSOLE_LOG_METHODS, method => {
     const original = console[method]
     console[method] = (...args) => {
-      client.leaveBreadcrumb('Console output', reduce(args, (accum, arg, i) => {
+      // if we are in an async context, use the client from that context
+      const c = client._clientContext && client._clientContext.getStore() ? client._clientContext.getStore() : client
```

Comment:
Would this be something we have to do in every plugin used in the node notifier now? If so it seems like we should make this automatic so that the plugins themselves don't need to change, otherwise we're inviting bugs where things are using the wrong client